apps/dialtone-documentation/docs/design/colors/themes/index.md

```diff
@@ -0,0 +1,209 @@
+- **Semantic Tokens**: Purpose-driven mappings (e.g., `--dt-color-foreground-primary`)  
+- **Theme Tokens**: Context-specific implementations (e.g., `--dt-theme-light-foreground-primary`)
+
+### Token Naming Convention
+
+Theme tokens follow a consistent naming pattern:
+
+```css
+--dt-theme-{theme-name}-{semantic-role}-{variant}
+```
+
+Examples:
+
+- `--dt-theme-light-surface-primary`
+- `--dt-theme-dark-foreground-secondary`
+- `--dt-theme-brand-accent-default`
+
+## Applying Tokens
+
+When implementing themes in your components, always use semantic tokens rather than base color values. This ensures your components automatically adapt when themes are switched.
+
+<dialtone-usage class="d-d-grid d-g24 d-g-cols2">
+  <template #do>
+    <div class="d-bgc-brand d-bar8">
+      <svg-loader class="d-fl1" name="theme-tokens-do" />
+    </div>
+    <h4>Use semantic theme tokens</h4>
+    Reference theme-aware semantic tokens that automatically adapt to the active theme context.
+  </template>
+  <template #dont>
+    <div class="d-bgc-brand d-bar8">
+      <svg-loader class="d-fl1" name="theme-tokens-dont" />
+    </div>
+    <h4>Don't use base color tokens directly</h4>
+    Avoid referencing base palette tokens directly as they won't respond to theme changes.
+  </template>
+</dialtone-usage>
+
+<div class="d-m32"></div>
+
+### CSS Implementation
+
+```css
+/* ✅ Correct - Uses theme-aware semantic tokens */
+.my-component {
+  background-color: var(--dt-theme-surface-primary);
+  color: var(--dt-theme-foreground-primary);
+  border-color: var(--dt-theme-border-default);
+}
+
+/* ❌ Incorrect - Uses base tokens that don't adapt to themes */
+.my-component {
+  background-color: var(--dt-color-black-100);
+  color: var(--dt-color-black-900);
+  border-color: var(--dt-color-black-300);
+}
+```
+
+## Customize Theming
+
+For advanced use cases, you can create custom themes by defining your own token mappings while maintaining the semantic structure.
+
+<div class="d-bgc-brand d-bar8"><svg-loader class="d-fl1" name="custom-themes" /></div>
+
+<div class="d-m32"></div>
+
+### Creating Custom Themes
+
+1. **Define your color palette**: Start with base color tokens that align with your brand
+2. **Map semantic meanings**: Assign base colors to semantic roles (primary, secondary, etc.)
+3. **Test accessibility**: Ensure contrast ratios meet WCAG guidelines
+4. **Validate consistency**: Check that your theme works across all component states
+
+### Custom Theme Example
+
+```css
+/* Custom brand theme */
+:root[data-theme="custom-brand"] {
+  --dt-theme-surface-primary: var(--dt-color-blue-50);
+  --dt-theme-surface-secondary: var(--dt-color-blue-100);
+  --dt-theme-foreground-primary: var(--dt-color-blue-900);
+  --dt-theme-foreground-secondary: var(--dt-color-blue-700);
+  --dt-theme-accent-default: var(--dt-color-gold-500);
+  --dt-theme-accent-hover: var(--dt-color-gold-600);
+}
+```
```

Comment:
Let's punt on doing this to a follow-up doc. We should discuss this more carefully.